demo/full/scripts/controllers/StickToLiveEdgeButton.jsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Button from "../components/Button.jsx";
+
+export default function StickToLiveEdgeButton({
+  isContentLoaded,
+  lowLatencyMode,
+  isStickingToTheLiveEdge,
+  changeStickToLiveEdge,
+}) {
+  return (isContentLoaded && lowLatencyMode) ?
```

Comment:
I don't think StickToTheLiveEdgeButton should be given that.

For me, it's just a dumb component (not controller) with only two props:
  - one indicating if it is enabled or not
  - one handling clicks

Knowing if it has to be displayed or not should be on the parent no?